client/layout/guided-tours/main-tour.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+
+import {
+	makeTour,
+	Tour,
+	Step,
+	Next,
+	Quit,
+	Continue,
+	Link,
+} from 'layout/guided-tours/config-elements';
+import {
+	isNewUser,
+	selectedSiteIsPreviewable,
+	selectedSiteIsCustomizable,
+	previewIsNotShowing,
+	previewIsShowing,
+} from 'state/ui/guided-tours/contexts';
+import Gridicon from 'components/gridicon';
+
+const context = state =>
+	true || isNewUser( state );
+
+export const MainTour = makeTour(
+	<Tour name="main" version="20160601" path="/" context={ context }>
```

Comment:
Added 250462c & 0a8fd6d. We can probably defer respecting the `path` until a later PR.